handlers/analytics.go

```diff
@@ -45,8 +46,9 @@ type AnalyticsLog struct {
 
 type AnalyticsLogEvent struct {
 	// Shared fields by all events
-	Type      string `json:"type"`
-	Timestamp int64  `json:"timestamp"`
+	Id        *string `json:"id"`
```

Comment:
lopri/nit, but by [go standards](https://go.dev/wiki/CodeReviewComments#initialisms) we keep acronyms fully uppercase (or fully lowercase). So these new fields should be `ID`